glide.yaml

```diff
@@ -33,14 +33,16 @@ import:
   repo:    https://github.com/openshift/kubernetes-apiextensions-apiserver.git
   version: origin-3.10-kubernetes-1.10.0
 - package: k8s.io/kubernetes
-  repo:    https://github.com/openshift/kubernetes.git
-  version: origin-3.10-kubernetes-1.10.0
+  repo:    https://github.com/liggitt/kubernetes.git
```

Comment:
This looks weird.  needs to be changed?

Reply:
Yes, waiting for bot run and will fixup deps